docker-compose.dev.yml

```diff
@@ -0,0 +1,72 @@
+version: '3.7'
+
+services:
+  # zookeeper:
+  #   image: wurstmeister/zookeeper
+  #   ports:
+  #     - "2181:2181"
+
+  # kafka:
+  #   image: wurstmeister/kafka:2.13-2.7.1
+  #   ports:
+  #     - "9092:9092"
+  #   links:
+  #     - zookeeper
+  #   environment:
+  #     KAFKA_ADVERTISED_HOST_NAME: 127.0.0.1
+  #     KAFKA_CREATE_TOPICS: "test:1:1"
+  #     KAFKA_ZOOKEEPER_CONNECT: zookeeper:2181
+  #     KAFKA_AUTO_CREATE_TOPICS_ENABLE: 'true'
+  #     KAFKA_MESSAGE_MAX_BYTES: 2000000
+  #   depends_on:
+  #     - zookeeper
+
+  rabbit:
+    build:
+      context: .
+      dockerfile: Dockerfile.test.yml
+    container_name: 'rabbitmq'
+    ports:
+      - "15674:15674"
+      - "15672:15672"
+
+  no_keys_service_db:
+    image: postgres:14-alpine
+    container_name: no_keys_service_db
+    restart: unless-stopped
+    environment:
+      - POSTGRES_DB=${KEYS_API_DB_NAME}
+      - POSTGRES_USER=${KEYS_API_DB_USER}
+      - POSTGRES_PASSWORD=${KEYS_API_DB_PASSWORD}
+    ports:
+      - ${KEYS_API_DB_PORT}:5432
+    volumes:
+      - ./.volumes/pgdata-${KEYS_API_CHAIN_ID}/:/var/lib/postgresql/data
+
+  no_keys_service_api:
+    image: lidofinance/lido-node-operators-keys-api:dev
+    container_name: no_keys_service_api
+    ports:
+      - '3001:3001'
+    environment:
+      - PORT=${KEYS_API_PORT}
+      - CORS_WHITELIST_REGEXP=${KEYS_API_CORS_WHITELIST_REGEXP}
+      - GLOBAL_THROTTLE_TTL=${KEYS_API_GLOBAL_THROTTLE_TTL}
+      - GLOBAL_THROTTLE_LIMIT=${KEYS_API_GLOBAL_THROTTLE_LIMIT}
+      - GLOBAL_CACHE_TTL=${KEYS_API_GLOBAL_CACHE_TTL}
+      - LOG_LEVEL=${LOG_LEVEL}
+      - LOG_FORMAT=${LOG_FORMAT}
+      - CHAIN_ID=${KEYS_API_CHAIN_ID}
+      - PROVIDERS_URLS=${RPC_URL}
```

Comment:
We walk in circles. I initially separated the application config from the Keys-API, then merged it. Now it is obvious that it would be easier to separate and then refactor this part in the context of a separate task.